src/collection.js

```diff
@@ -127,17 +127,17 @@ function importChange(transaction, remote) {
     // Not found locally but remote change is marked as deleted; skip to
     // avoid recreation.
     if (remote.deleted) {
-      return {type: "skipped", data: remote};
+      return {type: "skipped", data: remote, previous: local};
     }
     const synced = markSynced(remote);
     transaction.create(synced);
-    return {type: "created", data: synced};
+    return {type: "created", data: synced, previous: local};
   }
   const identical = deepEqual(cleanRecord(local), cleanRecord(remote));
   if (local._status !== "synced") {
     // Locally deleted, unsynced: scheduled for remote deletion.
     if (local._status === "deleted") {
-      return {type: "skipped", data: local};
+      return {type: "skipped", data: local, previous: local};
```

Comment:
What `previous` means here since we didn't do anything (we «skipped it» during importation) ? `data` is not enough ? (*the idea may be to have the same attributes everywhere I don't know*)